### ISSUE TITLE
AttributeError in CreateColourComposite algorithm

### ISSUE THREAD

**Issue by PedroVenancio** [alexbruy/processing-whitebox#18]
Hi @alexbruy 

I'm trying CreateColourComposite algorithm in QGIS 3.6 (Windows), but I'm not getting any output:

```
Processing algorithm…
Algorithm 'CreateColourComposite' starting…
Input parameters:
{ 'blue' : 'D:/orto_teste_split_b.tif', 'enhance' : True, 'green' : 'D:/orto_teste_split_g.tif', 'opacity' : None, 'output' : 'D:/orto_teste_split_composite.tif', 'red' : 'D:/orto_teste_split_r.tif', 'zeros' : False }

Traceback (most recent call last):
File "C:/Users/pedro.venancio/AppData/Roaming/QGIS/QGIS3\profiles\default/python/plugins\processing_whitebox\whiteboxAlgorithm.py", line 142, in processAlgorithm
arguments.append('--{}="{}"'.format(param.name(), os.path.normpath(layer.source())))
AttributeError: 'NoneType' object has no attribute 'source'

Execution failed after 0.22 seconds

Loading resulting layers
Algorithm 'CreateColourComposite' finished
```

Running it in CLI, it works:

`"C:\OSGeo4W64\WhiteboxTools_win_amd64\WBT\whitebox_tools.exe" --run=CreateColourComposite -v --red=D:/orto_teste_split_r.tif --green=D:/orto_teste_split_g.tif --blue=D:/orto_teste_split_b.tif -o=D:/orto_teste_split_composite.tif --enhance`

**Comment by alexbruy** [comment]
Fixed in 0.11.1

**Comment by PedroVenancio** [comment]
Perfect! Thank you very much @alexbruy !
